src/operator/tensor/elemwise_binary_op.h

```diff
@@ -474,6 +474,31 @@ class ElemwiseBinaryOp : public OpBase {
                                        std::vector<int> *in_attrs,
                                        std::vector<int> *out_attrs);
 
+  template<typename xpu, typename OP>
+  static void ComputeInt(const nnvm::NodeAttrs &attrs,
+                         const OpContext &ctx,
+                         const std::vector<TBlob> &inputs,
+                         const std::vector<OpReqType> &req,
+                         const std::vector<TBlob> &outputs) {
+    using namespace mxnet_op;
+    if (req[0] != kNullOp) {
```

Comment:
To save then indentation for the following lines, you can return here if `req[0]` is `kNullOp`.

Reply:
Doing this now for other functions as well in this file.